README.md

```diff
@@ -1,7 +1,7 @@
 # Pursuit-Core-Web-React-Introduction-Lab
 
 
-Recreate the image below using React and CSS.  Use CSS Grid Layout to arrange your components.
+Recreate the image below using React and CSS. Use CSS Grid Layout to arrange the four sections of the page.
```

Comment:
We should probably tell them where to put their files for each component (the tests will be looking for them in a certain place)

Reply:
Should we also point out in the README that the tests are going to be checking that the components are rendered in this exact structure?

Reply:
Yes, good calls!